avalon/tools/cbloader/widgets.py

```diff
@@ -103,6 +110,25 @@ def on_context_menu(self, point):
 
             action = QtWidgets.QAction(label, menu)
             action.setData((representation, loader))
+
+            # Add tooltip and statustip from Loader docstring
+            tip = inspect.getdoc(loader)
```

Comment:
I chose for `getdoc` because it also cleand up the docstring using `inspect.cleandoc`: https://docs.python.org/2/library/inspect.html#inspect.cleandoc